torchgeo/trainers/detection.py

```diff
@@ -58,37 +59,70 @@ class ObjectDetectionTask(LightningModule):
 
     .. code-block:: python
 
-        ['resnet18', 'resnet34', 'resnet50', 'resnet101', 'resnet152',
-        'resnext50_32x4d','resnext101_32x8d', 'wide_resnet50_2',
-        'wide_resnet101_2']
+       ['resnet18', 'resnet34', 'resnet50', 'resnet101', 'resnet152',
+       'resnext50_32x4d', 'resnext101_32x8d',
+       'wide_resnet50_2', 'wide_resnet101_2']
 
     .. versionadded:: 0.4
     """
 
-    def config_task(self) -> None:
-        """Configures the task based on kwargs parameters passed to the constructor."""
-        backbone_pretrained = self.hyperparams.get("pretrained", True)
```

Comment:
Previously this would use a pretrained backbone and download weights from the internet by default, which doesn't match the behavior of any other trainer.